packages/x-date-pickers-pro/src/DateRangePicker/DateRangePickerToolbar.tsx

```diff
@@ -27,18 +28,16 @@ const useUtilityClasses = (ownerState: DateRangePickerToolbarProps<any>) => {
 };
 
 export interface DateRangePickerToolbarProps<TDate>
```

Comment:
On the new pickers we will drop this `Omit` since the range pickers will also have a view management behavior to prepare for the `DateTimeRangePicker`